alexandria/core/tests/test_views.py

```diff
@@ -247,7 +253,7 @@ def test_validate_created_by_group(
         url = reverse(f"{model_name}-detail", args=[model.pk])
         # updates are not allowed to change created_by_group
         model.refresh_from_db()
-        assert model.created_by_group == "somegroup"
+        assert getattr(model, f"created_by_{attribute}") == "somegroup"
```

Comment:
does this hold for `attribute` == `user`

Reply:
Yes, changed the the name to be something more generic